### PR TITLE
✨ Fix #16: Adds command line argument `-c` to print the local emoji cache path.

### LINKED ISSUE #16: Add an argument to print emaji cache path
Add an argument to print emaji cache path so that in case of a problem one can use it to get the path instead of debugging the app.

### PR DESCRIPTION
**Pull request by rimbi**
Fix #16

### CODE REVIEW

**Review by zeenix — COMMENTED** [review]
Thanks so much for providing a PR, not only an issue. I'll have a look soon. 👍

Please make use of `--amend` option of `git` combined with `-f` option of `git push`, or just send fixes to your commit as additional commits (if there's only one commit originally) instead of creating multiple PRs.

That's a general advice. Otherwise you'll still get accused of PR spam. 😃

**Comment by rimbi** [comment]
Sorry for the inconvenience, you're right. It was due to a confusion on my local. I assumed I force pushed the changes but I did not see the effects on the PR changes.

File: src/main.rs [hunk 1]
```diff
@@ -31,6 +31,10 @@ struct Args {
     /// Run as git commit hook.
     #[arg(long, value_delimiter = ' ', num_args = 1..3)]
     hook: Vec<String>,
+
+    /// Prints the local emoji cache path.
```
Comment (zeenix) [inline src/main.rs:35]: `Prints` -> `Print`.

File: src/main.rs [hunk 2]
```diff
@@ -31,6 +31,10 @@ struct Args {
...
+    #[arg(short, long, default_value_t = false)]
+    print_emoji_cache_path: bool,
```
Comment (zeenix) [inline src/main.rs:37]: We don't need the name to be so descriptive. The help message does that for us. `cache_path` is sufficient.

**Review by zeenix — CHANGES_REQUESTED** [review]
LGTM me apart from the nitpicks. :) Since you're going to be modifying anyway, let me nitpick about the commit message too. :laughing: :

Ideally the commit message summary should be not more than 50 chars but never more than 74. That's why github is breaking the line. Since the reference to the issue being fixed is not the import part of the change, I'd suggest putting that in the body/description part:

```
✨ Add command line argument -p to print the local emoji cache path

Fix #16.
```

File: src/main.rs (hunk 2 above)
Comment (rimbi) [reply src/main.rs:37]: That's the way I usually do: long, descriptive names. But I will change it, no worries.

File: src/main.rs (hunk 1 above)
Comment (rimbi) [reply src/main.rs]: Done

**Comment by rimbi** [comment]
Review please :)

File: src/main.rs (hunk 2 above)
Comment (zeenix) [reply src/main.rs]: Yeah, me too but at the same time, not more than needed. E.g here it's obvious we're talking about `emoji` :)

**Review by zeenix — APPROVED** [review]
Thanks for addressing the comments. :+1:

File: src/main.rs (hunk 2 above)
Comment (rimbi) [reply src/main.rs]: but cutting `print` will cause degradation in the readability.

File: .gitignore [hunk 3]
```diff
@@ -1 +1,2 @@
 /target
+.vscode
```
Comment (zeenix) [inline .gitignore:2]: wait, just saw this now. Please split in a separate commit (not PR, just commit) as it's completely unrelated to this commit.

Comment (zeenix) [reply .gitignore:2]:
```sh
git reset HEAD^
git add -p # Only choose the .gitignore change
git commit 
git commit -a -C 05fcddb8e3b05140bec84c8f435c1e6fa53e5251
```

File: src/main.rs (hunk 2 above)
Comment (zeenix) [reply src/main.rs]: > but cutting print will cause degradation in the readability.

Don't think so since there are not that may possibilities here and behavior is the most obvious one.

Besides, people will likely find out this option through `--help` and that provides the description right next to it.

Anyway. Let's not bikeshed this to death. :laughing: If you still insist on `print-` prefix being important, do bring it back (but let's keep `-c` as the shortcut still).

File: .gitignore (hunk 3 above)
Comment (rimbi) [reply .gitignore:2]: Done

Comment (rimbi) [reply .gitignore]: Ah ok, coming...
